docs/sources/sharing/share-panel.md

```diff
@@ -1,67 +1,67 @@
+To share a direct link:
 
-## Direct Link Rendered Image
+1. Click **Copy**, this copies the default or the shortened URL to the clipboard.
+1. Send the copied URL to a Grafana user with authorization to view the link.
+1. You also optionally click **Direct link rendered image** to share an image of the panel.
 
-You also get a link to render a PNG image of the panel. Useful if you want to share an image of the panel. For more information about the requirements and how to configure image rendering, refer to [image rendering](/administration/image_rendering/).
+For more information, refer to the topic [Image rendering]({{< relref "../administration/image_rendering.md)" >}}).
 
-Example of a link to a server-side rendered PNG:
+Here is an example of a link to a server-side rendered PNG:
 
 ```bash
 https://play.grafana.org/d/000000012/grafana-play-home?orgId=1&from=1568719680173&to=1568726880174&panelId=4&fullscreen
 ```
-
 #### Query String Parameters For Server-Side Rendered Images
 
 - **width**: width in pixels. Default is 800.
 - **height**: height in pixels. Default is 400.
 - **tz**: timezone in the format `UTC%2BHH%3AMM` where HH and MM are offset in hours and minutes after UTC
 - **timeout**: number of seconds. The timeout can be increased if the query for the panel needs more than the default 30 seconds.
+- **scale**: numeric value to configure device scale factor. Default is 1. Use a higher value to produce more detailed images (higher DPI). Supported in Grafana v7.0+.
 
-## Embed Panel
+## Publish snapshots
+
+A panel snapshot shares an interactive panel publicly. Grafana strips sensitive data leaving only the visible metric data and series names embedded into your dashboard. Panel snapshots can be accessed by anyone with the link.
 
-You can embed a panel using an iframe on another web site. This tab will show you the html that you need to use.
+You can publish snapshots to your local instance or to [snapshot.raintank.io](http://snapshot.raintank.io). The latter is a free service provided by [Raintank](http://raintank.io), that allows you to publish dashboard snapshots to an external Grafana instance. You can optionally set an expiration time if you want the snapshot to be removed after a certain time period.
 
-> **Note:** This sharing requires [allow_embedding]({{< relref "../administration/configuration.md#allow-embedding" >}}) enabled and anonymous access, or proper configuration of the [cookie_samesite]({{< relref "../administration/configuration.md#cookie-samesite" >}}) setting.
+To publish a snapshot:
 
-Example:
+1. In the Share Panel dialog, click **Snapshot** to open the tab.
+1. Click on **Local Snapshot** or **Publish to snapshot.raintank.io**. This generates the link of the snapshot.
+1. Copy the snapshot link, and share it either within your organization or publicly on the Web.
+   
+If you created a snapshot by mistake, click **delete snapshot** to remove the snapshot from your Grafana instance.
```

Comment:
Is this capitalization for the bolded part correct? If so, then we should fix it in the UI.

Reply:
Documented as in the UI.